pkg/cache/cache.go

```diff
@@ -122,6 +124,7 @@ func (c *serviceAccountCache) addSA(sa *v1.ServiceAccount) {
 				resp.TokenExpiration = pkg.ValidateMinTokenExpiration(tokenExpiration)
 			}
 		}
+		c.saCount.Inc()
```

Comment:
addSA will get called every time a service account is updated, created, or even anytime the informer cache gets resynced (every 60s https://github.com/aws/amazon-eks-pod-identity-webhook/blob/019d1ba3137fc48ce5c75f1e5b0f1f5dd679aed7/main.go#L101) so this number will quickly become inaccurate. Any way to emit the metric at a lower layer (in set? )

Reply:
maybe it's easier to emit a metric for when the webhook performs a mutation?

Reply:
makes sense..made the change